static/js/create_edit.js

```diff
@@ -45,6 +45,17 @@ function txtUpdateOnChange() {
     shortLink.addEventListener("keyup", function () {
       let printout = document.getElementById("fueledEndPoint");
       printout.innerHTML = "/" + shortLink.value;
+      if (shortLink.value.includes("/")) {
+        document
+          .getElementsByClassName("formInput")[1]
+          .classList.add("border__red");
+        document.getElementsByClassName("invalid_endpoint")[0].style.display =
```

Comment:
use classes instead, this is easier, but not better. Imagine you will migrate this to react. Most probably you will scrap most of the JS but will be able to reuse styles.